pyFAI/units.py

```diff
@@ -57,15 +57,15 @@
 
 CONST_hc = hc = scipy.constants.c * scipy.constants.h / scipy.constants.e * 1e7
 """Product of h the Planck constant, and c the speed of light in vacuum
-in Angstrom.KeV. It is approximativly equal to:
- 
-pyFAI reference 12.398419292004204
-scipy v1.3.1:   12.398419739640717
-scipy-1.4.0rc1: 12.398419843320026
+in Angstrom.KeV. It is approximatively equal to:
+
+- pyFAI reference: 12.398419292004204
+- scipy v1.3.1:   12.398419739640717
+- scipy-1.4.0rc1: 12.398419843320026
 """
 
 CONST_q = scipy.constants.e
-"""One electron-volt is equal to 1.602176565⋅10-19 joules"""
+"""One electron-volt is equal to 1.602176634⋅10-19 joules"""
```

Comment:
Changed in [scipy 1.4.0](https://docs.scipy.org/doc/scipy/reference/release.1.4.0.html#scipy-constants-improvements)

Reply:
Thanks for updating it ...